src/main/java/org/wildfly/security/sasl/util/SetMechanismInformationSaslServerFactory.java

```diff
@@ -25,10 +25,11 @@
 import javax.security.sasl.SaslServer;
 import javax.security.sasl.SaslServerFactory;
 
-import org.wildfly.security._private.ElytronMessages;
 import org.wildfly.security.auth.callback.MechanismInformationCallback;
 import org.wildfly.security.auth.server.MechanismInformation;
 
+import static org.wildfly.security._private.ElytronMessages.log;
```

Comment:
Minor issue - statics should be first in the imports not last - can clean up later though.